msal/src/cache/java/com/microsoft/identity/client/BaseTokenCacheItem.java

```diff
@@ -31,13 +31,13 @@
 abstract class BaseTokenCacheItem {
 
     @SerializedName("client_id")
-    final String mClientId;
+    String mClientId;
```

Comment:
@weijjia If they are not initialized and no data comes back(no json), then it will be `null` anyways. You want me to add more code to just "make" them `null`?

@iambmelt @nazukj Based on the documentation example, they are not using final: https://static.javadoc.io/com.google.code.gson/gson/2.8.1/com/google/gson/annotations/SerializedName.html.

Also, if you finalize the parameters, I believe GSON is forced to use reflection like this:  https://stackoverflow.com/a/11220912/950427. 

I am trying to keep this pull request similar to the one I made for ADAL: https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/842